### PR TITLE
{AKS} `az aks create` | `az aks nodepool add/update`: Fix NSG control live tests and cmd description

### PR DESCRIPTION
**Pull request by lodrem**
**Related command**
``` bash
az aks create \
  --name ${CLUSTER_NAME} \
  --nodepool-asg-ids ${ASG_0_ID} ${ASG_1_ID} \
  --nodepool-allowed-host-ports 80/tcp 53/udp 4000-5000/tcp

az aks nodepool add \
  --name ${AGENTPOOL_NAME} \
  --asg-ids ${ASG_0_ID} ${ASG_1_ID} \
  --allowed-host-ports 80/tcp 53/udp 4000-5000/tcp

az aks nodepool update \
  --name ${AGENTPOOL_NAME} \
  --asg-ids ${ASG_0_ID} ${ASG_1_ID} \
  --allowed-host-ports 80/tcp 53/udp 4000-5000/tcp
```

**Description**
refer: #27900 

Fix the live tests and command descriptions.

**Testing Guide**

**History Notes**

- {AKS} `az aks create`: Fix live tests and command description.
- {AKS} `az aks nodepool add`: Fix live tests and command description.
- {AKS} `az aks nodepool update`: Fix live tests and command description.
---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [x] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [x] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

- [x] I adhere to the [Error Handling Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/error_handling_guidelines.md).

### CODE REVIEW

**Comment by yonzhan** [comment]
AKS

File: src/azure-cli/azure/cli/command_modules/acs/tests/latest/test_aks_commands.py [hunk 1]
```diff
@@ -9314,9 +9314,9 @@ def test_aks_create_with_nsg_control(self, resource_group, resource_group_locati
         asg_ids = [asg1['id'], asg2['id']]
 
         self.kwargs.update({
-            'asg1_id': asg1['id'],
-            'asg2_id': asg2['id'],
+            'asg_ids_argument': '--nodepool-asg-ids {}'.format(' '.join(asg_ids)),
```
Comment (FumingZhang) [inline src/azure-cli/azure/cli/command_modules/acs/tests/latest/test_aks_commands.py:9317]: Queued [live test](https://dev.azure.com/msazure/CloudNativeCompute/_build/results?buildId=83672330&view=results) to validate the change.

**Review by FumingZhang — APPROVED** [review]
LGTM

**Review by yanzhudd — APPROVED** [review]

**Comment by yanzhudd** [comment]
Now CLI is in a release process and PRs cannot be merged. Once release is complete, we would merge it.

**Pull request merged**